frontend/degree-plan/components/Dock/Dock.tsx

```diff
@@ -140,6 +152,11 @@ const Dock = ({ user, login, logout, activeDegreeplanId  }: DockProps) => {
     //     }
     // }, [isMount, dockedCourses]);
 
+    const { remove } = useSWRCrud<DockedCourse>(`/api/degree/docked`, { idKey: 'full_code' });
+    const handleRemoveCourse = (full_code: string) => {
```

Comment:
convention: `fullCode`

Reply:
<img width="1085" alt="Screenshot 2024-04-09 at 3 59 35 PM" src="https://github.com/pennlabs/penn-courses/assets/35184414/608ff4e5-899e-45f5-a436-f0d7ae5f5e4a">
👀

Reply:
yeah we've been using full_code. That might not be a good pattern (mb if so), but tbh I think this OK for now.

Reply:
Will defer to @esinx here, but if we want to clean this up, we should probably do so at the entire project level and likely after we solve the user facing issues